tuple/include/array_tuple_a_not_b.hpp

```diff
@@ -28,14 +28,27 @@
 
 namespace datasketches {
 
+/// array tuple A-not-B
 template<typename Array, typename Allocator = typename Array::allocator_type>
 class array_tuple_a_not_b: tuple_a_not_b<Array, Allocator> {
 public:
   using Base = tuple_a_not_b<Array, Allocator>;
   using CompactSketch = compact_array_tuple_sketch<Array, Allocator>;
 
+  /**
+   * Constructor
+   * @param seed for the hash function that was used to create the sketch
+   * @param allocator to use for allocating and deallocating memory
+   */
   explicit array_tuple_a_not_b(uint64_t seed = DEFAULT_SEED, const Allocator& allocator = Allocator());
 
+  /**
+   * Computes the a-not-b set operation given two sketches.
+   * @param a sketch A
+   * @param b sketch B
+   * @param ordered optional flag to specify if ordered sketch should be produced
+   * @return the result of a-not-b as a compact sketch
```

Comment:
Maybe A-not-B is more consistent with the input sketch names?

Reply:
Nevermind. Variables are lowercase. Maybe call them a and b in the above @param lines?